esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -431,7 +432,8 @@ void EsdlServiceImpl::configureUrlMethod(const char *method, IPropertyTree &entr
     entry.setProp("@prot", protocol);
     entry.setProp("@path", path);
 
-    Owned<ISmartSocketFactory> sf = createSmartSocketFactory(iplist, true);
+    Owned<ISmartSocketFactory> sf = new CSmartSocketFactory(iplist, true);
```

Comment:
That seems like an argument for leaving the code the way it was to me. If createSmartSocketFactory creates something other than a CSmartSocketFactory, it's because something (presumably a global setting) wanted Smart Sockets to use a different implementation (maybe creating an auditing version, for example). You would be bypassing such a mechanism here by making that change.